website/content/docs/nia/compatibility.mdx

```diff
@@ -1,35 +1,40 @@
 ---
 layout: docs
-page_title: Consul-Terraform-Sync Compatibility
+page_title: Consul-Terraform-Sync (CTS) Compatibility
 description: >-
-  Consul-Terraform-Sync Compatibility
+  Consul-Terraform-Sync (CTS) Compatibility
 ---
 
 # Compatibility
 
+The following tables list the Consul-Terraform-Sync (CTS) version compatibility for Consul, Terraform, and Vault.
+
 ## Consul
 
-Below are the supported Consul versions with compatible Consul-Terraform-Sync versions. The latest Consul-Terraform-Sync binary targets supporting the latest patch version of the three most recent Consul minor versions.
+Below are Consul-Terraform-Sync versions with supported Consul versions. The latest Consul-Terraform-Sync binary targets supporting the latest patch version of the three most recent Consul minor versions.
```

Comment:
I'm trying to understand the second sentence here, you're saying that CTS supports the latest patch version of the three most recent Consul minor versions... I'm wondering if there might be a better way to phrase that? Maybe:

`The latest Consul-Terraform-Sync binary target supports the three most recent Consul minor versions, along with their latest patch versions.`

Reply:
Also, and this is for the rest of the doc updates, did we want to go ahead and change the `Consul-Terraform-Sync` usage to `CTS` instead? I see you define the abbreviation at the top of the doc.